code/mission/missionbriefcommon.cpp

```diff
@@ -1681,7 +1681,7 @@ int brief_color_text_init(const char* src, int w, const char default_color, int
 	}
 
 	Assert(src != NULL);
-	n_lines = split_str(src, w, n_chars, p_str, BRIEF_META_CHAR);
+	n_lines = split_str(src, w, n_chars, p_str, MAX_BRIEF_LINE_LEN, BRIEF_META_CHAR, true);
```

Comment:
I don't think leading whitespace should be stripped.  Some briefings may use whitespace-aligned text.

Reply:
This, as this true is the default argument, would be a change to the current behaviour.
I agree with your reasoning, but should we really change established behaviour for this?

Reply:
I didn't realize `true` was default.  In that case, don't change established behavior, but also it's probably better not to add the optional argument.

Reply:
ok, cleaned that up